### PR TITLE
Put goals and objectives under little collapsible sections too!

### LINKED ISSUE #988: Build out the cleaned up Needs and Objectives section
As discussed in #943, the Needs and Objectives section needs a little bit of finetuning to match the styles of the other sections.

image

(Related: I think @quarterback is working on updating the content for the help text for this section)

### PR DESCRIPTION
**Pull request by mgwalker**
Closes #988 

### This pull request changes...
- The goals and objectives under the "Performance Measures and Benchmarks" section are under the happy little blue collapsible sections just like personnel/non-personnel/contractor expenses.

### This pull request is ready to merge when...
- [ ] Tests have been updated (and all tests are passing)
- [ ] This code has been reviewed by someone other than the original author

### This feature is done when...
- [ ] Design has approved the experience
- [ ] Product has approved the experience

### CODE REVIEW

**Comment by mgwalker** [comment]
@lauraponce 👍 Should the x in the blue box be completely removed, or just when the goal is expanded?

And should we also update the expense sections to behave the same way?  (They all use the same blue bar element now, so if they need to behave differently, that's fine I just need to make sure to account for that.  😄)

**Comment by lauraponce** [comment]
Good catch on the expenses section! Yes, let's remove that extra X there also

**Comment by brendansudol** [comment]
bless you for abstracting the mini-header component!

File: web/src/containers/activity/Goals.js [hunk 1]
```diff
@@ -9,66 +9,149 @@ import {
...
+        {goals.length === 0 ? (
+          <NoDataMsg>{t('activities.expenses.noDataNotice')}</NoDataMsg>
+        ) : (
+          <div className="mt3 pt3 border-top border-grey">
+            {goals.map((goal, i) => (
+              <div key={goal.key}>
+                <MiniHeader
+                  content={
+                    <div className="col-11 truncate">
+                      {plaintext(goal.description)}
```
Comment (brendansudol) [inline web/src/containers/activity/Goals.js]: what are your thoughts on using this `plaintext` method vs. the `Md` component here?

Comment (mgwalker) [reply web/src/containers/activity/Goals.js]: What would happen if the start of the description was an H1?  I'm open to giving it a try and seeing what happens - the `plaintext` method doesn't exactly fill me with joy.  😆

Comment (brendansudol) [reply web/src/containers/activity/Goals.js]: ohh, that's a good point -- alright, plaintext works for me then!

**Pull request merged**